app/controllers/MovementsController.scala

```diff
@@ -124,21 +124,22 @@ class MovementsController @Inject()(
   def putArrival(arrivalId: ArrivalId): Action[NodeSeq] = authenticateForWrite(arrivalId).async(parse.xml) {
```

Comment:
Do we need _putArrival_? We're in the movements controller so _put_ might be enough to indicate we're creating a new movement?

Reply:
I think that at the time we didn't spilt the controllers by resource so it made sense then, but you're right it might be unnecessary now, I'll probably raise a PR to rename that